query/src/main/scala/filodb/query/exec/RemoteExec.scala

```diff
@@ -47,6 +51,29 @@ trait RemoteExec extends LeafExecPlan with StrictLogging {
                 querySession: QuerySession)
                (implicit sched: Scheduler): ExecResult = ???
 
+  private def applyTransformers(resp: QueryResponse,
```

Comment:
Huge callout: we now apply transformations to `RemoteExec` plans. This is needed until the `hasDescendantAggregateOrJoin` pushdown problem is solved ([example](https://github.com/filodb/FiloDB/blob/5b84fadc2975112b4d3af779d609f77adc22d67f/coordinator/src/test/scala/filodb.coordinator/queryplanner/PlannerHierarchySpec.scala#L3524)).

Reply:
should we add this context to the function documentation as well  ?

Reply:
Good call 👍